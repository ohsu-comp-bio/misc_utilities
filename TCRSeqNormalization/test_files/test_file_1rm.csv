Seq. Count,Percent,N Sequence,"CDR3 Length, nt.",AA Sequence,V segments,J segments
100,0.04725898,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ1-1
300,0.10417769,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV2,TRBJ1-2
500,0.2362949,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV2,TRBJ1-1
10,0.004725898,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV3,TRBJ1-3
1,0.0004725898,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV4,TRBJ1-1
40,0.01890359,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-1
1000,0.4725898,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-2
15,0.007088847,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-4
150,0.07088847,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV3,TRBJ3-1
