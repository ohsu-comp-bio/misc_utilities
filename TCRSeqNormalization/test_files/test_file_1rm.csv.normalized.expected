Seq..Count,Percent,N.Sequence,CDR3.Length..nt.,AA.Sequence,V.segments,J.segments
100,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ1-1
300,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV2,TRBJ1-2
500,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV2,TRBJ1-1
10,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV3,TRBJ1-3
1,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV4,TRBJ1-1
40,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-1
1000,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-2
15,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV1,TRBJ2-4
150,0.100425736,TGTGCCAGCAGTCTGTATGAGCAGTACTTC,30,CASSLYEQYF,TRBV3,TRBJ3-1
